plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -815,6 +815,13 @@ fu_cros_ec_usb_device_write_firmware (FuDevice *device,
 								     firmware,
 								     section,
 								     error)) {
+				if ((*error)->code == G_USB_DEVICE_ERROR_NOT_SUPPORTED) {
+					g_debug ("failed to transfer section, trying another write, ignoring error: %s",
+						 (*error)->message);
```

Comment:
ew. Can you add `g_autoptr(GError) error_local = NULL` and then use that instead? We don't actually enforce that `error != NULL` and this would be explotastic.

Reply:
g_error_matches

Reply:
Thanks. Uploaded a new version.